README.md

```diff
@@ -60,10 +73,10 @@ nats section must look like below
   
 ## Environmet variables
 
-Service will look for config.toml first if not found it will  be configured 
-with env variables and new config.toml will be saved with values populated from env vars.
-The service is configured using the environment variables presented in the
-following table. Note that any unset variables will be replaced with their
+Service will look for `config.toml` first and if not found it will be configured   
+with env variables and new `config.toml` will be saved with values populated from env vars.  
+The service is configured using the environment variables presented in the  
+following table. Note that any unset variables will be replaced with their  
```

Comment:
What are this white spaces at the end of the line?

Reply:
https://gist.github.com/shaunlebron/746476e6e7a4d698b373